internal/v1/handler_blueprints_test.go

```diff
@@ -34,20 +34,14 @@ type BlueprintExportResponseUnmarshal struct {
 	SnapshotDate   *string                                       `json:"snapshot_date,omitempty"`
 }
 
-func makeTestServer(t *testing.T, apiSrv *string, csSrv *string) (dbase db.DB, srvURL string, shutdown func(t *testing.T)) {
+func makeTestServer(t *testing.T, apiSrv *string) (dbase db.DB, srvURL string, shutdown func(t *testing.T)) {
 	srv := startServer(t, &testServerClientsConf{
 		ComposerURL: func() string {
 			if apiSrv != nil {
 				return *apiSrv
 			}
 			return ""
 		}(),
-		CSURL: func() string {
```

Comment:
Same comment as above, the commit message and the content seem to not align here. Maybe just add something like `and move csSrv directly into testServer struct` or something.